spec/mini-lo-dash-spec.js

```diff
@@ -156,6 +156,60 @@
 
         });
 
+        describe('#find', function () {
+            it('returns the matched element', function () {
+                var matchedElement
+
+                var catsWithColors = [
+                    {
+                        name: 'Harry',
+                        color: 'brown'
+                    },
+                    {
+                        name: 'Julia',
+                        color: 'black'
+                    },
+                    {
+                        name: 'Anton',
+                        color: 'gray'
+                    }
+                ]
+
+                matchedElement = _.find(catsWithColors, function(cat) {
```

Comment:
shouldn't we expect the matched element to be: `{name: 'Anton', color: 'gray'}`?